scripts/check-rng-hygiene.sh

```
@@ -0,0 +1,145 @@
+        next
+      }
+      if (line ~ /^#[[:space:]]*include/) next
+      if (index($0, optout) || blockopt) { blockopt = 0; next }
+      # Match against code with string literals blanked, so a log message that
+      # says "pseudo-random" is not itself a finding. Report the original line.
+      code = $0
+      gsub(/"[^"]*"/, "\"\"", code)
+      if (code ~ pat) printf "%s:%d:%s\n", FILENAME, FNR, $0
+      blockopt = 0
+    }
+  ' 2>/dev/null || true
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)